tests/tfm/tfm_psa_test/README.rst

```diff
@@ -12,6 +12,9 @@ The TF-M platform security architecture test sample provides a basis for validat
 Requirements
 ************
 
+This test sample requires that the device is provisioned with the PSA root-of-trust security parameters using the :ref:`provisioning image <provisioning_image>` sample.
```

Comment:
This one only applies for the CONFIG_TFM_PSA_TEST_ATTESTATION=y case.
Should we specify that?

It's only possible to run one of the test-suite at a  time, and if you're not running the attestation suite this can be skipped.